docs/ai/get-started-mcp.md

```diff
@@ -0,0 +1,100 @@
+
+MCP client and server can exchange a set of standard messages:
+
+| Message             | Description |
+|---------------------|-------------|
+| `InitializeRequest` | This request is sent by the client to the server when it first connects, asking it to begin initialization. |
+|`ListToolsRequest`     |  Sent by the client to request a list of tools the server has.       |
+|`CallToolRequest`     |  Used by the client to invoke a tool provided by the server.       |
+|`ListResourcesRequest`     | Sent by the client to request a list of available server resources.        |
+|`ReadResourceRequest`     |  Sent by the client to the server to read a specific resource URI.       |
+|`ListPromptsRequest`     | Sent by the client to request a list of available prompts and prompt templates from the server.       |
+|`GetPromptRequest`     |  Used by the client to get a prompt provided by the server.       |
+|`PingRequest`     |   A ping, issued by either the server or the client, to check that the other party is still alive.      |
+|`CreateMessageRequest`     |  A request by the server to sample an LLM via the client. The client has full discretion over which model to select. The client should also inform the user before beginning sampling, to allow them to inspect the request (human in the loop) and decide whether to approve it.       |
+|`SetLevelRequest`     | A request by the client to the server, to enable or adjust logging.        |
+
+## Develop with the MCP C# SDK
+
+As a .NET developer, you can use MCP by creating MCP clients and servers to enhance your apps with custom integrations. MCP reduces the complexity involved in connecting an AI model to various tools, services, and data sources.
+
+The official [MCP C# SDK](https://github.com/modelcontextprotocol/csharp-sdk) is available through NuGet and enables you to build MCP clients and servers for .NET apps and libraries. The SDK is maintained through collaboration between Microsoft, Anthropic, and the MCP open protocol organization.
+
+To get started, add the MCP C# SDK to your project:
+
+```dotnetcli
+dotnet add package ModelContextProtocol --prerelease
+```
+
+Instead of building unique connectors for each integration point, you can often leverage or reference prebuilt integrations from various providers such as GitHub and Docker:
+
+- [Available MPC clients](https://modelcontextprotocol.io/clients)
+- [Available MCP servers](https://modelcontextprotocol.io/examples)
+
+### Integration with Microsoft.Extensions.AI
+
+The MCP C# SDK depends on the [Microsoft.Extensions.AI libraries](/dotnet/ai/ai-extensions) to handle various AI interactions and tasks. These extension libraries provides core types and abstractions for working with AI services, so developers can focus on coding against conceptual AI capabilities rather than specific platforms or provider implementations.
```

Comment:
```suggestion
The MCP C# SDK depends on the [Microsoft.Extensions.AI libraries](ai-extensions.md) to handle various AI interactions and tasks. These extension libraries provides core types and abstractions for working with AI services, so developers can focus on coding against conceptual AI capabilities rather than specific platforms or provider implementations.
```